libs/cardano-ledger-core/src/Cardano/Ledger/Plutus/CLI.hs

```diff
@@ -0,0 +1,54 @@
+module Cardano.Ledger.Plutus.CLI where
+
+import Cardano.Ledger.Binary (Version, mkVersion64)
+import Cardano.Ledger.Plutus.Language (Language)
+import Data.Int (Int64)
+import Numeric.Natural (Natural)
+import Options.Applicative
+
+data Opts = Opts
+  { optsScript :: !String
```

Comment:
The mandatory argument to the binary is not just a script, but a script with full context. So, we want to clearly indicate it in the CLI. Also it could be useful to provide an option for overriding the script itself
```suggestion
  { optsScriptWithContext :: !String
  , optsScript :: !(Maybe String)
```